features/AssetCategory.feature

```diff
@@ -284,4 +283,102 @@ Feature: AssetCategory
     Then The document "engine-ayse":"asset-category":"littleTruck" content match:
       | parent | null |
 
+  Scenario: Use metadata enum type and link with assetCategory
+    When I successfully execute the action "device-manager/assetCategory":"create" with args:
+      | engineId  | "engine-ayse"  |
+      | body.name | "coloredTruck" |
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse"          |
+      | body.name      | "color"                |
+      | body.valueType | "string"               |
+      | body.valueList | ["red","blue","green"] |
+      | body.mandatory | true                   |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
```

Comment:
It feel strange to me to fill the param `metadataId` despite the fact that it was the `name` when is was created

Reply:
If I understand well, you use the metadata name as the document ID right?

If it's the case then my comment does not applies, I thought it was different values. You are right to use the ID to ensure uniqueness :+1: